release_notes/v2.0.0-beta.md

```diff
@@ -0,0 +1,191 @@
+
+In prior releases, the GetHistoryForKey chaincode API had no
+guarantees on the order of returned results.
+Starting in Fabric v2.0, the GetHistoryForKey chaincode API
+will return results from newest to oldest in terms of ordered transaction
+height (block height and transaction height within block).
+This will allow applications to iterate through the top results
+to understand recent changes to a key.
+
+**FAB-16722: The 'provisional' genesis method of generating the system channel
+for orderers has been removed.**
+
+Existing users of the provisional genesis method
+should instead set BootstrapMethod to 'file', and generate a genesis block file
+using configtxgen. Orderer nodes will then use the generated file for the
+orderer system channel.
+
+**FAB-16477 and FAB-17116: New configuration for orderer genesismethod and genesisfile**
+
+The orderer config `general.genesismethod` and `general.genesisfile` are replaced
+by the new `general.bootstrapmethod` and `general.bootstrapfile`.
+
+**FAB-15343: System Chaincode Plugins have been removed.**
+
+As part of a general
+move away from go plugins as an extension mechanism for Fabric, the ability to
+add system chaincodes via go plugins has been removed.  Users wishing to extend
+Fabric with custom system chaincodes may rebuild the peer binary with the
+system chaincode built into the binary.  This system chaincode should then be
+defined and initialized like any other user chaincode would be.  This new model
+is very similar to the plugin model (which required that the plugin to be built
+at the same exact commit of Fabric), and addresses the significant shortcomings
+around the lifecycle and validation of system chaincode transactions.
+
+**FAB-11096: Docker images with Alpine Linux**
+
+Hyperledger Fabric Docker images will now use Alpine Linux,
+a security-oriented, lightweight Linux distribution.
+
+**FAB-11096: Bash not available in Docker images with Alpine Linux**
+Bash is no longer available in Fabric images. Utilize Alpine's built-in
+sh or ash instead.
+
+**FAB-16987: Go version has been updated to 1.13.4.**
+
+
+Deprecations
+------------
+
+**FAB-15754: The 'Solo' consensus type is deprecated.**
+
+The 'Solo' consensus type has always been marked non-production and should be in
+use only in test environments, however for compatibility it is still available,
+but may be removed entirely in a future release.
+
+**FAB-16408: The 'Kafka' consensus type is deprecated.**
+
+The 'Raft' consensus type was introduced in v1.4.1 and has become the preferred
+production consensus type.  There is a documented and tested migration path from
+Kafka to Raft, and existing users should migrate to the newer Raft consensus type.
+For compatibility with existing deployments, Kafka is still supported,
+but may be removed entirely in a future release.
+
+**FAB-7559: Support for specifying orderer endpoints at the global level
+in channel configuration is deprecated.**
+
```

Comment:
Perhaps add: "To fit scenarios where orderers are run by different organizations, you should specify orderer endpoints at the orderer organization level, which uses only the TLS CA certificates of that organization, in contrast to the global channel level endpoints which use the aggregation of all orderer TLS CA certificates across all orderer organizations"

Reply:
done